.pre-commit-config.yaml

```diff
@@ -0,0 +1,69 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+files: |
+    (?x)(
+        noxfile\.py|
+        setup\.py|
+        docs\/.+\.py|
+        benchmarks\/.+\.py|
```

Comment:
This will be the location of benchmarks if/when we add them. Keeping the line minimises the differences with `.pre-commit-config.yaml` in other repos.